tasmota/tasmota_xsns_sensor/xsns_91_vindriktning.ino

```diff
@@ -96,9 +107,64 @@ bool VindriktningReadData(void) {
 
 /*********************************************************************************************/
 
+void VindriktningWriteData(uint8_t cmd_len, const uint8_t *cmd_data) {
+
+  uint8_t tx_buffer[5];
+  int len = 0;
+  int i = 0;
+  uint8_t sum = 0;
+
+  if (cmd_len > 2) {
```

Comment:
In such a simple case where only one command exists, do we need all this code right now?

Reply:
As there is just a single command to request PM data from the sensor PM1006, I simplified the code as you propose.